scripts/run-e2e-config.ts

```diff
@@ -152,3 +152,13 @@ export const yarn2Cra: Parameters = {
     `yarn dlx create-react-app@{{version}} {{name}}-v{{version}}`,
   ].join(' && '),
 };
+
+export const reactInYarnWorkspace: Parameters = {
+  name: 'reactInYarnWorkspace',
+  version: 'latest',
+  generator: [
+    'cd {{name}}-v{{version}}',
+    'echo "{ \\"name\\": \\"workspace-root\\", \\"private\\": true, \\"workspaces\\": [] }" > package.json',
```

Comment:
Init a Yarn workspace with the most simple way (i.e. without any tricks or new dependency)